lib/web/fetch/formdata-parser.js

```diff
@@ -365,13 +370,53 @@ function parseMultipartFormDataName (input, position) {
   // - `%0A`: 0x0A (LF)
   // - `%0D`: 0x0D (CR)
   // - `%22`: 0x22 (")
-  name = name
+  name = new TextDecoder().decode(name)
     .replace(/%0A/i, '\n')
     .replace(/%0D/i, '\r')
     .replace(/%22/, '"')
```

Comment:
can this even be correct? 

This code is actually just replacing the first occurence because it uses replace and not replaceAll and the regex is not using the global flag.

```suggestion
    .replace(/%0A/gi, '\n')
    .replace(/%0D/gi, '\r')
    .replace(/%22/g, '"')
```